src/Dofs/DofHandler.jl

```diff
@@ -152,8 +162,20 @@ function close!(dh::DofHandler{dim}) where {dim}
     nextdof = 1 # next free dof to distribute
     push!(dh.cell_dofs_offset, 1) # dofs for the first cell start at 1
 
+    # Assumes that all nodes have the same number of dofs
```

Comment:
This is a fundamental assumption in the whole code that `ndofs_per_node = sum(dh.field_dims)`, since one cannot add fields to a select set of nodes as far as I can tell. I was just being explicit.